### PR TITLE
indent: use 4 spaces

### PR DESCRIPTION
**Pull request by alagoutte**

### CODE REVIEW

**Comment by nmbradford** [comment]
Jenkins run tests please.

**Comment by nmbradford** [comment]
Jenkins run tests please.

**Comment by nmbradford** [comment]
Jenkins run tests please.

File: PowerNSX.psm1 [hunk 1]
```diff
@@ -171,10 +171,9 @@ https://github.com/vmware/powernsx/issues
             public internalWebResponse() {
                 this.Headers = new Dictionary<string,string>(StringComparer.OrdinalIgnoreCase);
             }
-	    public override string ToString()
-	    {
-		return this.Content;
-	    }
+        public override string ToString() {
```
Comment (dcoghlan) [inline PowerNSX.psm1]: This needs to be indented one more level

File: PowerNSX.psm1 [hunk 2]
```diff
@@ -171,10 +171,9 @@ https://github.com/vmware/powernsx/issues
...
+            return this.Content;
```
Comment (dcoghlan) [inline PowerNSX.psm1]: Same as above

**Comment by alagoutte** [comment]
Update the patch ! (and rebase)

**Comment by dcoghlan** [comment]
Jenkins test this please

**Comment by dcoghlan** [comment]
Jenkins test this please

**Comment by dcoghlan** [comment]
Jenkins test this please

**Comment by alagoutte** [comment]
May be need a update to fix conflict ?

**Pull request closed without merging**